src/beacons/use-beacons.ts

```diff
@@ -166,14 +167,14 @@ const requestAndroidPermissions = async () => {
   );
 
   if (locationStatus !== RESULTS.GRANTED) {
-    return false;
+    return true;
```

Comment:
According to the Figma, if this is granted or not, then stop and continue with the onboarding, and is true because the Bluetooth permission was granted.